integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -95,7 +94,7 @@ def _serialize_items(self, channel_metadata_items):
             sort_keys=True
         ).encode('utf-8')
 
-    def _transmit_create(self, channel_metadata_item_map, content_updated_mapping):
+    def _transmit_create(self, channel_metadata_item_map):
```

Comment:
nit: these `_transmit_*()` methods are now extremely similar to each other and use the same method signature.  Maybe make a base method and use a `functools.partial` or something to factor this down even more?  e.g.
```python
def _transmit_action(self, item_map, client_method, final_action, action_name):
    # ...do all the work, eventually calling client_method(serialized_chunk)
    # and final_action(chunk, response_status_code, response_body)
    # all the exception handling, etc.
    # use e.g. action_name='delete' for your logging messages.

self._transmit_create = functools.partial(
    self._transmit_action, 
    self=self, 
    client_method=self.client.create_content_metadata, 
    final_action=self._create_transmissions,
    action_name='create',
)
self._transmit_update = functools.partial(
    self._transmit_action, 
    self=self, 
    client_method=self.client.update_content_metadata, 
    final_action=self._update_transmissions,
    action_name='update'
)
self._transmit_delete = functools.partial(
    self._transmit_action, 
    self=self, 
    client_method=self.client.delete_content_metadata, 
    final_action=self._delete_transmissions,
    action_name='delete'
)
```